internal/nodes/avi_model_l4_translator.go

```diff
@@ -375,6 +375,45 @@ func PopulateServers(poolNode *AviPoolNode, ns string, serviceName string, ingre
 	return pool_meta
 }
 
+func PopulateServersForMultiClusterIngress(poolNode *AviPoolNode, ns, cluster, serviceNamespace, serviceName string, key string) []AviPoolMetaServer {
+	var servers []AviPoolMetaServer
+	svcName := generateMultiClusterKey(cluster, serviceNamespace, serviceName)
+	success, siNames := objects.SharedMultiClusterIngressSvcLister().MultiClusterIngressMappings(ns).GetSvcToSI(svcName)
+	if !success {
+		utils.AviLog.Warnf("key: %s, msg: failed to get service imports mapped to service with name: %v", key, svcName)
+		return servers
+	}
+
+	for _, siName := range siNames {
+		serviceImport, err := utils.GetInformers().ServiceImportInformer.Lister().ServiceImports(ns).Get(siName)
+		if err != nil {
+			utils.AviLog.Errorf("key: %s, msg: failed to get service imports with name: %v", key, siName)
```

Comment:
Should we change this to warn ? The user may not create backend service  in some cases

Reply:
I have changed it to a warning.